### DISCUSSION TITLE
Wrong axis label position

### DISCUSSION THREAD

**Discussion by shockhs** [airbnb/visx#1426] [Q&A]
Hi all! 👋

How can I fix the bug with the Label positioning at the axes in XYChart. The demo from the documentation has the same bug. If Y values are too big, the labels overlap each other

![image](https://user-images.githubusercontent.com/59527678/151114624-1034aa12-0fd1-4ae3-8398-3992ff935077.png)

**Comment by rishabhjaiz** [comment]
Couple of suggestions.
1. You can adjust the chart margin so you have more space on the right side.
2. You can also truncate the large number to more readable string like 2k, 4k, .. etc this will free you some space.

**Reply by Ding-Fan** [reply]
An example of `margin` of `XYChart` is

```jsx
<XYChart
  margin={
    {
      top: 60,
      right: 60,
      bottom: 60,
      left: 60
    }
  }
>
// ...
```

It took me hours to find out the proper way to use it :face_with_spiral_eyes:

**Comment by williaster** [comment]
this isn't a "bug" so I converted it to a discussion. @rishabhjaiz 's suggestions are great, also noting some related discussions:
- https://github.com/airbnb/visx/discussions/1188
- https://github.com/airbnb/visx/discussions/1414
